pest/src/span.rs

```diff
@@ -127,18 +127,6 @@ impl<'i> Span<'i> {
     }
 
     /// Captures a slice from the `&str` defined by the `Span`.
-    ///
```

Comment:
This should probably also not be deleted since it's user-facing.

Reply:
I'll come up with a better example here as well